lib/ui/viewer/file_details/upload_icon_widget.dart

```diff
@@ -83,11 +85,18 @@ class _UpdateIconWidgetState extends State<UploadIconWidget> {
           final bool isIgnored = snapshot.data!;
           final bool isQueuedForUpload =
               !isIgnored && widget.file.collectionID != null;
+          if (isQueuedForUpload) {
+            return Padding(
+              padding: const EdgeInsets.all(2.0),
+              child: CircularProgressIndicator(
```

Comment:
This will keep showing up when to there's no network/upload is not in progress or user is out of storage/active sub.